duckduckgo/iOS
# Autofill prompt icon + copy updates

Task/Issue URL: https://app.asana.com/0/72649045549333/1205786958985456/f
Tech Design URL:
CC:

**Description**:
Removes the website favicon + url at the top of the autofill prompts and add app icon to the top of all autofill prompts. Also updates copy to replace the word `logins` with `passwords`



**Steps to test this PR**:
1. Visit https://fill.dev/form/login-simple and enter any username & password
2. Confirm the `Save a Login` prompt has the same layout & copy as this:
<img width=200 src="https://github.com/duckduckgo/iOS/assets/91189922/d3904798-9ced-4219-92ac-e016715c3f70">

3. Tap `Save Password`
4. Go back to https://fill.dev/form/login-simple and confirm the `Use Saved Login` prompt has the same layout & copy as this: 
<img width=200 src="https://github.com/duckduckgo/iOS/assets/91189922/76599cd6-c5f4-4539-8858-82482627e48f">

5. On https://fill.dev/form/login-simple enter the username of your saved login, but enter a different password
6. Confirm the `Update Password` prompt has the same layout & copy as this: 
<img width=200 src="https://github.com/duckduckgo/iOS/assets/91189922/f4e44b0d-64a3-4bc2-a7cc-94e8365c6dd4">

7. Go to https://news.ycombinator.com/login and in the login form enter a dummy password and tap 'login'
8. Confirm the `Save Password` prompt has the same layout & copy as this: 
<img width=200 src="https://github.com/duckduckgo/iOS/assets/91189922/e29cf7e7-9e6c-487c-8d57-1861caea4b18">

9. Now enter a dummy username in the same form and tap 'login'
10. Confirm the `Update Username` prompt has the same layout & copy as this:  
<img width=200 src="https://github.com/duckduckgo/iOS/assets/91189922/116a2154-f372-4703-8caf-c499e8e7e366">

11. Visit https://fill.dev/form/registration-email and tap the key icon in the password field
12. Confirm the `Generated Password` prompt has the same layout & copy as this:  
<img width=200 src="https://github.com/duckduckgo/iOS/assets/91189922/a00cd71f-a97c-4a51-858b-5d6530e9c2d8">

13. Review the translated strings, confirming 4 have been modified and 2 (now unused) are deleted



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
